addons/web_editor/static/src/xml/grid_layout.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="web_editor.background_grid">
```

Comment:
Let's not indent the first level but add blank lines around the template definition (after `<templates>` and before `</templates>`)